src/app/components/StatsWidget.tsx

```diff
@@ -0,0 +1,121 @@
+"use client";
+
+import { useAppSelector } from "hooks";
+import { fetchKpiData, KpiData } from "kpis/kpis-utils";
+import { useEffect, useState } from "react";
+import { simpleFormatNumber } from "utils/simpleFormatNumber";
+import AnimatedCounter from "./AnimatedCounter";
+
+const StatsWidget = () => {
+  // google sheet data
+  const [kpiData, setKpiData] = useState<KpiData | null>(null);
+  // alphadex data
+  const { pairsList } = useAppSelector((state) => state.rewardSlice);
+
+  // fetch socials & trade volumes (in XRD / USD)
+  useEffect(() => {
+    (async () => {
+      try {
+        const data = await fetchKpiData();
+        setKpiData(data);
+      } catch (error) {
+        console.error("StatsWidget -> fetchKpiData error", error);
+      }
+    })();
+  }, []);
+
+  // 1. google sheet data
+  // 1.1. socials data
+  const socials = kpiData?.socials;
+  const numOfIgFollowers = socials?.instagramFollowers ?? 0;
+  const numOfXFollowers = socials?.twitterFollowers ?? 0;
+  const numOfUTubeFollowers = socials?.youtubeSubscribers ?? 0;
+  const totalNumOfFollowers =
+    numOfIgFollowers + numOfXFollowers + numOfUTubeFollowers;
+
+  // 1.2. trade volume data - total & weekly - XRD & USD
+  const tradeVolume = kpiData?.tradeVolume;
+
+  // total
+  const tradeVolumeTotalXrd = tradeVolume?.total?.XRD ?? 0;
+  const tradeVolumeTotalUsd = tradeVolume?.total?.USD ?? 0;
+
+  // weekly
+  const tradeVolumeWeeklyXrd = tradeVolume?.weekly?.XRD?.at(-1)?.value ?? 0;
+  const tradeVolumeWeeklyUsd = tradeVolume?.weekly?.USD?.at(-1)?.value ?? 0;
+
+  // 2. alphadex data
+  // Number of pairs listed
+  const numOfPairs = pairsList.length;
+
+  return (
+    <div className="flex opacity-80 flex-col gap-y-8 mx-auto justify-center items-center my-8">
```

Comment:
Please add a lot more margin at the bottom, I suggest at least `mb-32`

![image](https://github.com/user-attachments/assets/5d606e8e-ff0e-44c5-b1b4-c6d5f6b84afe)

Reply:
Yes, I added mb-32